### PR TITLE
Clarify nature of remote providers

### PR DESCRIPTION
**Pull request by eccentricexit**
# TLDR 

The docs wrongly claim that tools such as infura and pockt are light clients. This PR fixes that.

# Description

## Remote Providers

A light client:
- Is software that you run locally;
- Is Connected to the p2p network;
- Performs local validation on data received.

When using a remote provider, the user must trust that it won't serve it bad data as there is no local validation at all.
This is specially critical as there is no guarantee on the software that is serving on used the truested endpoint.

In addition, a name resolution compromise can send requests to a malicious endpoint to perform an attack.

## Pokt

Pokt network makes use of a trusted centralized service to route requests. This means that an adversary can, for example, compromise the `pokt.network` domain and lie to everyone trusting the service or just bring all nodes offline (from the point of view of the users).

### CODE REVIEW

**Comment by jclapis** [comment]
Thanks for the update!
